### ISSUE TITLE
time() function used incorrectly in badgeos_user_meets_points_requirement()

### ISSUE THREAD

**Issue by tw2113** [opencredit/badgeos#486]
As is, we have
```
// If the user just earned the badge, though, don't let them earn it again
// This prevents an infinite loop if the badge has no maximum earnings limit
if ( $last_activity >= time('-2 seconds') )
	$return = false;
```
but time() does not have parameters able to be passed in. We should be doing the -2 seconds part separate from fetching the timestamp.
```
$timeminus2 = time() - 2;
if ( $last_activity >= $timeminus2 ) {
	$return = false;
}
```

Originally reported at https://wordpress.org/support/topic/automatically-award-based-on-points?replies=23#post-7530008

**Comment by tw2113** [comment]
https://github.com/opencredit/badgeos/tree/486-time_fix

**Comment by tw2113** [comment]
Fixed in 1.4.7 release
